schema/metrics/README.md

```diff
@@ -0,0 +1,124 @@
+
+A structured schema provides a clear, consistent format, making it easier for observability tools to process and aggregate the data.
+This in turn makes it easier for engineers to understand the performance and behavior of their systems, and quickly identify potential issues.
+
+When metrics are unstructured, it can be difficult for observability tools to extract meaningful information from them. 
+For example, if the data for a particular metric is not consistently recorded in the same format, it can be difficult to compare and analyze performance data over time.
+Similarly, if metrics are not consistently named or categorized, it can be difficult to understand their context and significance.
+
+With a structured schema in place, observability tools can automatically extract and aggregate data, making it easier to understand system behavior at a high level. 
+This can help teams quickly identify performance bottlenecks, track changes in system behavior over time, and make informed decisions about system performance optimization.
+
+## Details
+The next section provides the Simple Schema for Observability support which conforms with the OTEL specification.
+
+- metrics.mapping presents the template mapping for creating the Simple Schema for Observability index
+- metrics.schema presents the json schema validation for verification of a metrics document conforms to the mapping structure
+
+## Metrics
+see [OTEL metrics convention](https://opentelemetry.io/docs/reference/specification/metrics/)
+see [OTEL metrics protobuf](https://github.com/open-telemetry/opentelemetry-proto/tree/main/opentelemetry/proto/metrics/v1)
+
+Simple Schema for Observability conforms with OTEL metrics protocol which defines the next data model:
+
+### data-stream
+[data-stream](https://opensearch.org/docs/latest/opensearch/data-streams/) Data streams simplify this process and enforce a setup that best suits time-series data, such as being designed primarily for append-only data and ensuring that each document has a timestamp field.
+A data stream is internally composed of multiple backing indices. Search requests are routed to all the backing indices, while indexing requests are routed to the latest write index.
+
+As part of the Observability naming scheme, the value of the data stream fields combine to the name of the actual data stream :
+
+`{data_stream.type}-{data_stream.dataset}-{data_stream.namespace}`.
+This means the fields can only contain characters that are valid as part of names of data streams.
+
+- **type** conforms to one of the supported Observability signals (Traces, Logs, Metrics, Alerts)
+- **dataset** user defined field that can mainly be utilized for describing the origin of the signal
+- **namespace** user custom field that can be used to describe any customer domain specific classification
+
+
+If nothing is stated in the namespace / dataset - the signal information would be routed into the default data-stream indices
```

Comment:
what is default data-stream indices? does it created by plugin?

Reply:
the plugin creates mapping template for all the Observability signals (trace,metrics,logs) and creates the default data_stream for each see https://github.com/opensearch-project/observability/issues/1405

Reply:
default index will not be created on loading - only template